app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+from os import makedirs, remove
+
+
+def move_file(command: str) -> None:
+    list_with_info = command.split(" ")
+    file_in_name = list_with_info[1]
+    dirs_name = ""
+
+    if list_with_info[0] != "mv":
+        return
+    try:
```

Comment:
also, you should add a condition if you don't have dirs in the command just use `os.rename()`